pkg/deployment/network.go

```diff
@@ -30,16 +30,19 @@ import (
 func ConfigureNetwork(ctx context.Context, helper *helper.Helper, obj client.Object, sshKeySecret string, inventoryConfigMap string) error {
 
 	role := ansibleeev1alpha1.Role{
-		Name:     "edpm_network_config",
-		Hosts:    "all",
-		Strategy: "linear",
+		Name:        "edpm_network_config",
+		Hosts:       "all",
+		Strategy:    "linear",
```

Comment:
should this be `free`? 
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/edpm_ansible/playbooks/deploy-edpm-network-configure.yml#L19

Reply:
No, I'd prefer to keep the strategy as linear. It makes things much easier to debug when looking at the ansible log. Unless we need to optimize, I'd say stick with linear.